client/src/components/Tabs/index.js

```diff
@@ -1,12 +1,26 @@
-import React from "react";
+import React, { Component } from "react";
 
 import "./tabs.css";
 
-const Tabs = ({ className }) => (
-  <div className="tabs-style">
-    <div className={className}>Flight route</div>
-    <div className={className}>Map</div>
-  </div>
-);
-
+class Tabs extends Component {
+  goToInfo = () => {
+    window.location = "/flightinfo";
```

Comment:
If you use the `history.push()` method to change pages, react-router will route the user without reloading the page.  I think you have access to it at `this.props.history.push()`